frontend/assets/stylesheets/components/_elevated-banner.scss

```diff
@@ -0,0 +1,96 @@
+/* ==========================================================================
+   Elevated Banner
+   ========================================================================== */
+.elevated-banner {
+  @supports (clip-path : polygon(0% 100%, 0% 0%, 100% 0%, 100% 95.73%)) {
```

Comment:
Thank you for spotting that @Ap0c. In Firefox and IE, we will fallback to the default behaviour (show no diagonal). In the future, a solution could be replace the polygon with an SVG. http://stackoverflow.com/questions/33816793/clip-path-doesnt-work-in-firefox-values